windows-installer/fluent-template.conf

```diff
@@ -29,7 +29,7 @@
 <match **>
   @type google_cloud
   buffer_type file
-  buffer_path /var/log/google-fluentd/buffers
+  buffer_path /var/run/google-fluentd/buffers
```

Comment:
We probably should set it to something else, like `C:\temp\google-fluentd\buffers` or even `C:\ProgramData\google-fluentd\buffers`. In fact, we use `WIN_EVT_POS_FILE_PLACE_HOLDER` for the pos file, and substitute that at installation time. Why aren't we doing this for buffer files?

Reply:
Good idea. Done.